content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1112,6 +1123,780 @@ And create it:
 kubectl apply -f my-crontab.yaml
 crontab "my-new-cron-object" created
 ```
+<!--
+## Validation rules
+-->
+## 验证规则
```

Comment:
Please add `{#anchor}` so that ` 当[验证规则特性](#validation-rules)被启用` can refer to here.

Ref: [Localization Guild](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E5%85%B3%E4%BA%8E%E9%93%BE%E6%8E%A5)

Reply:
There seem to be other places where anchors are used but not explicitly defined.